content/2023/configurations/banners.tsx

```diff
@@ -14,11 +14,11 @@ const showBannerPageConfigs: ShowBannerPageConfigType[] = [
       title: 'sciwork conference',
       subTitle:
         'December 9th to 10th\nNational Yang Ming Chiao Tung University',
-      children: (
-        <Button to={REGISTER_URL} target="_blank" variant="action">
-          Register
-        </Button>
-      ),
+      // children: (
```

Comment:
I do not know about frontend coding.  But as a general rule of thumb it is not a good style to use comments for disabling dead code in production.

Reply:
It is not necessary to keep code. Removed at 4dd77e72507c8a8e17446a50b68bc6cde5727a25